### PR TITLE
Hausdorff distance

### PR DESCRIPTION
**Pull request by josteinbf**
This PR implements the Hausdorff distance. Docs, examples and tests are incomplete, but the functionality seems to be ok.

Some discussion has already taken place here: #738.

Ping @jni @JDWarner

### CODE REVIEW

**Comment by joefutrelle** [comment]
@jni are there still plans to merge this PR?

**Comment by JDWarner** [comment]
Yes @joefutrelle, we'd love to have this feature!

**Comment by stefanv** [comment]
Someone still needs to adop the PR, though.

File: skimage/measure/_set_metrics.pyx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    for i in range(points_sup.shape[0]):
+        d2_min_i = MAX_FLOAT64
+        for j in range(points_inf.shape[0]):
+            d2_j = 0.
+            for k in range(points_sup.shape[1]):
+                d2_j += (points_sup[i, k] - points_inf[j, k])**2
+            d2_min_i = d2_min_i if d2_min_i < d2_j else d2_j
+        d2_max = d2_max if d2_max > d2_min_i else d2_min_i
+
```
Comment (jni) [inline skimage/measure/_set_metrics.pyx]: @emmanuelle I haven't found a standard algo but you are absolutely correct! I'll add this suggestion to the PR. =)

**Comment by stefanv** [comment]
@jni Special achievement unlocked: necromancer.  Inline commenting on a 3 year old PR!

**Comment by jni** [comment]
@stefanv that's genius. Github should definitely gamify this sort of stuff. =D

**Comment by alexandrejaguar** [comment]
Hi @scikit-image/core, IMHO this is something we'd like to maintain. How're going to solve this one?
Thanks!

**Comment by clementkng** [comment]
Hi @alexandrejaguar, if adopting/closing this PR is mainly a manner of addressing the feedback above, I'd like to attempt it. I'm a newcomer but I'm very interested in contributing!

**Comment by sciunto** [comment]
Superseeded. See the new PR.

**Pull request closed without merging**